sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/e2e/application-exception.aest.getPendingApplicationExceptions.e2e-spec.ts

```diff
@@ -0,0 +1,114 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import { ApplicationExceptionStatus } from "@sims/sims-db";
+import { getUserFullName } from "../../../../utilities";
+import { TestingModule } from "@nestjs/testing";
+import { saveFakeApplicationWithApplicationException } from "../application-exception-helper";
+
+describe(`ApplicationExceptionAESTController(e2e)-getPendingApplicationExceptions`, () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let appModule: TestingModule;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource, module } =
+      await createTestingAppModule();
+    app = nestApplication;
+    appDataSource = dataSource;
+    appModule = module;
+  });
+
+  it("Should get pending application exceptions when available.", async () => {
+    // Arrange
+    const application1Promise = saveFakeApplicationWithApplicationException(
```

Comment:
IMO if the expectation of the test case is to validate the logic filtering the exception with pending status, then one pending and another with non-pending status alone must be sufficient here.